From 33e778ac7fd1ef858e5a183ed5e3a8057ce02608 Mon Sep 17 00:00:00 2001
From: Pragnesh Patel <pragnesh.patel@sifive.com>
Date: Tue, 19 May 2020 15:43:34 +0530
Subject: [PATCH 1/1] efi_loader: Remove unnecessary debug

Remove unnecessary debug() from efi_set_variable_common().
native_name is NULL, so there is no meaning to print it.

Signed-off-by: Pragnesh Patel <pragnesh.patel@sifive.com>
Reviewed-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_variable.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/efi_loader/efi_variable.c b/lib/efi_loader/efi_variable.c
index 10892684d1..9698efbc5d 100644
--- a/lib/efi_loader/efi_variable.c
+++ b/lib/efi_loader/efi_variable.c
@@ -883,8 +883,6 @@ static efi_status_t efi_set_variable_common(u16 *variable_name,
 	u32 attr;
 	efi_status_t ret = EFI_SUCCESS;
 
-	debug("%s: set '%s'\n", __func__, native_name);
-
 	if (!variable_name || !*variable_name || !vendor ||
 	    ((attributes & EFI_VARIABLE_RUNTIME_ACCESS) &&
 	     !(attributes & EFI_VARIABLE_BOOTSERVICE_ACCESS))) {
-- 
2.26.2

